source/description.rst

```diff
@@ -960,12 +961,12 @@ Same as for groups. See :numref:`sec-quantity` for details.
 ``linkable``
```

Comment:
I'm not sure if we are using the ``linkable`` feature anywhere. This may be something we could eventually deprecate.

Reply:
I agree